src/main/java/com/sprint/mission/discodeit/service/ChannelService.java

```diff
@@ -0,0 +1,15 @@
+package com.sprint.mission.discodeit.service;
+
+import com.sprint.mission.discodeit.entity.Channel;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+public interface ChannelService {
+    Channel create(Channel channel);
+    Channel read(UUID id);
+    List<Channel> readAll();
+    Channel update(UUID id, Channel updatedChannel);
```

Comment:
첫번째 인자 id 값이 채널의 ownerId이니 인자명을 좀더 명확히 해도 좋을 것 같아요.
```suggestion
    Channel update(UUID userId, Channel updatedChannel);
```